### DISCUSSION TITLE
Welcome to react-native-naver-map Discussions!

### DISCUSSION THREAD

**Discussion by QuadFlask** [QuadFlask/react-native-naver-map#54] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dongnaedavid** [comment]
Is there any way to control default value for tilt? We want to be able to load the map in tilt by default. 
![image](https://user-images.githubusercontent.com/72372580/112242149-9c45d900-8c8e-11eb-8732-bf79797f10cf.png)

**Reply by QuadFlask** [reply]
try this
```js
<NaverMapView tilt={tiltValue}>
// ....
</NaverMapView>
```

or 

```js
<NaverMapView camera={{tilt: tiltValue}}>
// ....
</NaverMapView>
```

**Reply by dongnaedavid** [reply]
![image](https://user-images.githubusercontent.com/72372580/112246206-9ef7fc80-8c95-11eb-9cfa-046367ff2282.png)
 :(
